packages/react/src/contexts/treasure.tsx

```diff
@@ -131,6 +141,21 @@ const TreasureProviderInner = ({
     [chain.id],
   );
 
+  const onAuthTokenUpdated = (authToken: string) => {
```

Comment:
This should likely be memoized with `useCallback` since it's in the `useLauncher` hook's `useEffect` dependency list